.github/workflows/release-embedding-sdk.yml

```diff
@@ -86,9 +84,23 @@ jobs:
         with:
           m2-cache-key: "release-sdk"
 
+      - name: Update readme
+        run: |
+          ./bin/embedding-sdk/release_utils.bash update_readme ${{ inputs.sdk_version }}
+
       - name: Bump published npm package version
         run: |
-          sed -i -E 's/("version": ").*"/\1${{ inputs.sdk_version }}"/' enterprise/frontend/src/embedding-sdk/package.template.json
+          ./bin/embedding-sdk/release_utils.bash update_package_json_template ${{ inputs.sdk_version }}
+
+      - name: Upload release utils script, so it can be used later on when checking out `master` branch
+        uses: actions/upload-artifact@v4
+        with:
+          name: release-util
+          path: ./bin/embedding-sdk/release_utils.bash
+
+      - name: Update changelog
```

Comment:
I do this now for 2 reasons
1. We should generate the change log with the specified commit. Not when checking out `master` in the later job.
2. This way we ensure CHANGELOG.md is included in the npm package.